### PR TITLE
add guards v1.5.1 with autoupdate config

### LINKED ISSUE #7109: [Request] Add guards.js
**Library name: guards.js
**Git repository url: https://github.com/on-site/guards.js
**npm package url(optional):** 
**License(s): MIT license
**Official homepage: http://guardsjs.com/
**Wanna say something? Leave message here: **

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by dannyxx001**
@PeterDaveHello please help me check this pull request, thank you.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/on-site/guards.js
 * Official website (optional, not the repository): http://guardsjs.com/
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 23
   - Count of watchers: 40
   - Count of forks: 3
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #7109, cc @smellsblue

### CODE REVIEW

File: ajax/libs/guards/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  "homepage": "http://guardsjs.com/",
+  "docs": "http://guardsjs.com/documentation.html",
+  "demo": "http://guardsjs.com/",
+  "download": "http://guardsjs.com/downloads.html",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/on-site/guards.js.git",
+    "basePath": "",
+    "files": [
+      "downloads/guards*"
```
Comment (PeterDaveHello) [inline ajax/libs/guards/package.json]: If you use `downloads` here, it must exist in your commit.

Comment (dannyxx001) [inline ajax/libs/guards/package.json]: Sorry, I forgot to do it. I have updated it, please help me check again, thanks.

**Comment by smellsblue** [comment]
For the checklist, I'm the author of `guards.js` and am more than happy to see `guards.js` hosted at cdnjs.

Is there anything in particular you need from me?

File: ajax/libs/guards/package.json (hunk 1 above)
Comment (PeterDaveHello) [inline ajax/libs/guards/package.json]: @dannyxx001 it's better to take `downloads` as `basePath`, so that we don't need to add that folder.

**Comment by PeterDaveHello** [comment]
Hi @smellsblue,

It'll be great if you can put a link in the readme/website about it's been hosted on CDNJS once we finished this PR. Thanks.

File: ajax/libs/guards/package.json (hunk 1 above)
Comment (dannyxx001) [inline ajax/libs/guards/package.json]: OK, I have  updated it.

File: ajax/libs/guards/package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    "jquery": "~1.4.4"
+  },
+  "description": "Guard your forms with class (or any other selector).",
+  "keywords": [
+    "validation"
+  ],
+  "homepage": "http://guardsjs.com/",
+  "docs": "http://guardsjs.com/documentation.html",
+  "demo": "http://guardsjs.com/",
+  "download": "http://guardsjs.com/downloads.html",
```
Comment (PeterDaveHello) [inline ajax/libs/guards/package.json]: Please remove `docs`, `demo`, and `download`;

File: ajax/libs/guards/package.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    "email": "stonemj+jqueryplugins@gmail.com",
+    "url": "http://reasonnumber.com/"
+  },
+  "license": "MIT",
+  "dependencies": {
+    "jquery": "~1.4.4"
+  },
+  "description": "Guard your forms with class (or any other selector).",
+  "keywords": [
+    "validation"
```
Comment (PeterDaveHello) [inline ajax/libs/guards/package.json]: Let's try to add some keywords here.

Comment (dannyxx001) [inline ajax/libs/guards/package.json]: OK, I have update. Please help me check again, Thanks.

**Comment by PeterDaveHello** [comment]
You didn't remove the `download` folder.

**Comment by dannyxx001** [comment]
Sorry, I'm silly. XD I have updated it

**Pull request merged**